### PR TITLE
Ear 943 mbs fixes

### PR DESCRIPTION
**Pull request by MattElcock**
### What is the context of this PR?
More MBS changes. check the Jira ticket for the spec.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: data/en/mbs_9106.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
                         "type": "Question",
```
Comment (MattElcock) [inline data/en/mbs_9106.json:87]: This file needs the mandatory:false from the others in it

File: data/en/mbs_9158.json [hunk 2]
```diff
@@ -209,10 +203,10 @@
                                     "id": "answer20459962-662d-4439-9987-2b89b0ac5479to",
                                     "type": "Date",
                                     "mandatory": true,
-                                    "label": "To",
+                                    "label": "To:",
                                     "q_code": "12",
                                     "maximum": {
-                                        "meta": "ref_p_end_date",
+                                        "value": "2016-06-12",
```
Comment (MattElcock) [inline data/en/mbs_9158.json]:
```suggestion
"meta": "ref_p_end_date",
```

File: data/en/mbs_9161.json [hunk 3]
```diff
@@ -211,7 +211,7 @@
                                     "id": "answer5b169b72-cdd2-47ee-9d68-53ab102b3524to",
                                     "type": "Date",
                                     "mandatory": true,
-                                    "label": "To",
+                                    "label": "To:",
```
Comment (MattElcock) [inline data/en/mbs_9161.json]:
```suggestion
                                    "label": "To",
```

File: data/en/mbs_9201.json [hunk 4]
```diff
@@ -98,7 +98,7 @@
                         "type": "Question",
```
Comment (MattElcock) [inline data/en/mbs_9201.json:98]: This file needs the mandatory false like the others

**Review by MattElcock — COMMENTED** [review]
All banners need to have

```
<b>Include</b>: furloughed employees in any employment figures.<br><b>
```
in them

File: data/en/mbs_9251.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "eq_id": "mbs",
-    "form_type": "0251",
+    "form_type": "0253",
```
Comment (MattElcock) [inline data/en/mbs_9251.json]:
```suggestion
    "form_type": "0251",
```

File: data/en/mbs_9255.json [hunk 6]
```diff
@@ -6,596 +6,746 @@
     "data_version": "0.0.1",
```
Comment (MattElcock) [inline data/en/mbs_9255.json:6]: Pannels need to be added here.

**Comment by MattElcock** [comment]
> <b>Include</b>: furloughed employees in any employment figures.<br>

I went back through and added this.

**Comment by MattElcock** [comment]
> All banners need to have
> 
> ```
> <b>Include</b>: furloughed employees in any employment figures.<br><b>
> ```
> 
> in them

I went back through and added this.

**Comment by MattElcock** [comment]
There's a few questions dotted around that have been set to required. Previously, the business accidentally did this and making them optional was one of the things they wanted to fix in this.

I've asked about if they've all been done and raised that there's still some lurking.

**Review by martyncolmer — APPROVED** [review]

**Review by jasehumphr — APPROVED** [review]

**Pull request merged**